projects/README.md

```diff
@@ -2,16 +2,15 @@
-|[HDF-GROUP](http://www.hdfgroup.org 'The HDF Group')|[open](https://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING 'HDF5 License')|HDF5 is a data model, library, and file format for storing and managing data|1.8.16|none|[diff](https://github.com/distributePro/ovsrpro/blob/master/patches/hdf5.patch 'patch/diff')|
-|[Kerberos](http://web.mit.edu/kerberos/ 'Kerberos')|[open](http://web.mit.edu/kerberos/krb5-1.14/doc/mitK5license.html 'MIT Kerberos License information')|Kerberos is a network authentication protocol (only built on Windows)|5.1.14|none|none|
-|[librdkafka](https://github.com/edenhill/librdkafka 'librdkafka on github')|[open](https://github.com/edenhill/librdkafka/blob/master/LICENSE '2-clause BSD license')|librdkafka is a C library implementation of the Apache Kafka protocol, containing both Producer and Consumer support|0.9.5|[repo](https://github.com/distributePro/librdkafka 'distributePro fork of librdkafka repo on github')|[diff](https://github.com/distributePro/librdkafka/compare/edenhill:v0.9.5...xp-v0.9.5 'patch/diff') [win-diff](https://github.com/distributePro/ovsrpro/blob/master/patches/librdkafka-windows.patch 'patch/diff (Windows only)')|
-|[NOVAS](http://aa.usno.navy.mil/software/novas/novas_info.php 'NOVAS')|[open](http://aa.usno.navy.mil/software/novas/novas_c/README.txt '(See Section IV. Using NOVAS in Your Applications)')|NOVAS is an integrated package of ANSI C functions for computing many commonly needed quantities in positional astronomy.|3.1|none|[diff](https://github.com/distributePro/ovsrpro/blob/master/patches/novas.patch 'patch/diff')|
+|[HDF-GROUP](http://www.hdfgroup.org 'The HDF Group')|[open](https://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING 'HDF5 License')|HDF5 is a data model, library, and file format for storing and managing data|1.8.16|none|none|
+|[librdkafka](https://github.com/edenhill/librdkafka 'librdkafka on github')|[open](https://github.com/edenhill/librdkafka/blob/master/LICENSE '2-clause BSD license')|librdkafka is a C library implementation of the Apache Kafka protocol, containing both Producer and Consumer support|v0.9.5|[repo](https://github.com/distributePro/librdkafka 'distributePro fork of librdkafka repo on github')|[diff](https://github.com/distributePro/librdkafka/compare/edenhill:v0.9.5...xp-v0.9.5 'patch/diff')|
+|[NOVAS](http://aa.usno.navy.mil/software/novas/novas_info.php 'NOVAS')|[open](http://aa.usno.navy.mil/software/novas/novas_c/README.txt '(See Section IV. Using NOVAS in Your Applications)')|NOVAS is an integrated package of ANSI C functions for computing many commonly needed quantities in positional astronomy.|3.1|none|none|
```

Comment:
Why was Kerberos removed?

Reply:
@brobeson for some reason when I run cmake, this file changes. I don't know why. This is the results

Reply:
Did you disable building Kerberos and cppzmq? I _think_, the readme is updated as part of the build process, so if something isn't built, it's removed.

Reply:
This README will get regenerated at build time - but I've avoided committing the one that is produced just because on Linux we don't build some of the pieces that are built on Windows and it will leave them out of this file.  There are also a couple of diffs that are input manually for some of the projects and those will also be removed if you commit this file...

Reply:
the README shouldn't change based on what platform you are on -- there are simply *many* issues with how the ovrsrpro projects have been done...

Reply:
the `projects/README.md` file is generated at cmake time (not build time) -- but only if the `XP_STEP` is *not* `STREQUAL "build"` -- you can see that every step calls `xpMarkdownReadmeFinalize()` *except* `build` in the `proExecuteStep()` macro https://github.com/smanders/externpro/blob/18.01.1/modules/macpro.cmake#L196-L218